locale/foreman.pot

```diff
@@ -16714,6 +16685,314 @@ msgstr ""
 msgid "Connected to: %s"
 msgstr ""
 
+#: action_names.rb:2
```

Comment:
I think you ran extraction with plugins (tasks / Katello?) enabled, so it also included all those tasks. Sadly foreman-tasks can't separate them so you should run extraction without plugins.

Reply:
I think it's sufficient to disable all gems in the short term, then re-enable after it.

Reply:
I'm not sure why, but disabling the gems + the katello config file is still resulting in Katello stuff making it into the `.pot` file.